.env.example

```diff
@@ -0,0 +1,2 @@
+AZURE_AI_MISTRAL_LARGE_ENDPOINT=MISTRAL_LARGE_ENDPOINT
+AZURE_AI_MISTRAL_LARGE_KEY=API_KEY
```

Comment:
Nit pick but every file should terminate with a new line

Some reasons it's bad are [explained here](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Although I've never encountered an issue myself it's good practice to do this for any file